functional-tests/src/test/groovy/pages/app/MapPage.groovy

```diff
@@ -6,8 +6,8 @@ class MapPage extends BaseAppPage {
   static at = { searchBarText.equals("Mine name or location") }
   static url = "/map"
   static content = {
-    searchBarText { $("esri-map .esri-ui-top-left.esri-ui-corner form input").attr("placeholder") }
+    // searchBarText { $("esri-map .esri-ui-top-left.esri-ui-corner form input").attr("placeholder") }
```

Comment:
Remove if not needed.

Reply:
Yep, removed. Left them in by accident while updating the tests.